features/admin.applications.v1/components/forms/general-details-form/general-details-form.tsx

```diff
@@ -980,8 +982,7 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
                 loading={ isSubmitting }
                 label={ t("common:update") }
                 hidden={
-                    (isSubOrganizationType && !isDiscoverable) ||
-                    !hasRequiredScope || (
+                    isSubOrganizationType || !hasRequiredScope || (
```

Comment:
Hope omitting `isDiscoverable` intentional.

Reply:
Yes this was added by me and forgot to remove from previous PR